_scripts/audit.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
+
+FILE=.last-audit
+NOW=$(date +%s)
+
+function run_audit() {
+  npm run audit
+  echo $NOW > $FILE
+}
+
+if [ -f "$FILE" ]; then
+  THRESHOLD=86400 # 24 hours
+  EXISTING=$(< $FILE)
+  DIFFERENCE=$(($NOW - $EXISTING))
+
+  if [ "$DIFFERENCE" -ge "$THRESHOLD" ]; then
+    run_audit
+  else
+    echo "Skipping audit..."
```

Comment:
nit: something about why it was skipped. maybe "Thetan levels nominal. Come back tomorrow for another audit" j/k